### DISCUSSION TITLE
sVB Kid Programmer, Level 2, Variables ...

### DISCUSSION THREAD

**Discussion by boa2145** [VBAndCs/sVB-Small-Visual-Basic#95] [General]
Thank you, Mohammad, for publishing the second eBook for "Small Visual Basic", Kid Programmer! I think it's great that you're taking care of a series of programming books for "Small Visual Basic", since up until now there was only the big manual for sVB. This Kindle edition is all about the important topic of "variables", what they are, how they are managed dynamically and how they are used in programs.

[photostudio_1733071831845.jpg](https://github.com/user-attachments/assets/50badd8e-31c3-4182-a99a-adf1d4dc4e44)

**Comment by VBAndCs** [comment]
Hi Gregor,
You are welcome J
And to be the first to know, the 3rd book "Hi Turtle!" is finished and is under review!
This is a part from its introduction:
What we will learn in this book?

  *   This book is designed to improve coding skills, through repeated practice with turtle drawings that are similar in most parts. Such repetition will help kids memorize the syntax, get used to calling methods, passing parameters, reading properties, and storing values in variables.

It is important for kids to write each code sample by hand to achieve these goals.

  *   The book also introduces some new programming skills:

  *   Using the turtle for drawing.
  *   Changing colors via code.
  *   Defining subroutines and parameters.
  *   Calling subroutines and passing arguments.
  *   Using loops to repeat code execution.

  *   The book aslo aims to enhance Math skills by:

  *   Introducing some Geomerty basics, like calculating the perimeter and area of squares, rectangles and circles.
  *   Introducing basic properties of some polygons, such as triangles and quadrilaterals.
  *   Drawing more than 20 different geometic figures by just moving and turing the turle arround, withiout needing any trigonometric calcualtions.
  *   Genetating rdndom numbers and colors, and using them in interesting allplications.
  *   Rounding decimal numbers.
  *   Converting lengthes measured in pixels to centimeters.


There is a lot of fun to have with the turtle, and much knowledge to gain in this 100-pages book.
So, what are you waiting for?
Let’s get started!

Hi Gregor,
You are welcome J
And to be the first to know, the 3rd book "Hi Turtle!" is finished and is under review! I expect to publish it in 2 days.
This is a part from its introduction:
What we will learn in this book?

  *   This book is designed to improve coding skills, through repeated practice with turtle drawings that are similar in most parts. Such repetition will help kids memorize the syntax, get used to calling methods, passing parameters, reading properties, and storing values in variables.

It is important for kids to write each code sample by hand to achieve these goals.

  *   The book also introduces some new programming skills:

  *   Using the turtle for drawing.
  *   Changing colors via code.
  *   Defining subroutines and parameters.
  *   Calling subroutines and passing arguments.
  *   Using loops to repeat code execution.

  *   The book aslo aims to enhance Math skills by:

  *   Introducing some Geomerty basics, like calculating the perimeter and area of squares, rectangles and circles.
  *   Introducing basic properties of some polygons, such as triangles and quadrilaterals.
  *   Drawing more than 20 different geometic figures by just moving and turing the turle arround, withiout needing any trigonometric calcualtions.
  *   Genetating rdndom numbers and colors, and using them in interesting allplications.
  *   Rounding decimal numbers.
  *   Converting lengthes measured in pixels to centimeters.


There is a lot of fun to have with the turtle, and much knowledge to gain in this 100-pages book.
So, what are you waiting for?
Let’s get started!



________________________________
From: Gregor ***@***.***>
Sent: Sunday, December 1, 2024 5:00 PM
To: VBAndCs/sVB-Small-Visual-Basic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [VBAndCs/sVB-Small-Visual-Basic] sVB Kid Programmer, Level 2, Variables ... (Discussion #95)


Thank you, Mohammad, for publishing the second eBook for "Small Visual Basic", Kid Programmer! I think it's great that you're taking care of a series of programming books for "Small Visual Basic", since up until now there was only the big manual for sVB. This Kindle edition is all about the important topic of "variables", what they are, how they are managed dynamically and how they are used in programs.

photostudio_1733071831845.jpg<https://github.com/user-attachments/assets/50badd8e-31c3-4182-a99a-adf1d4dc4e44>

**Reply by boa2145** [reply]
Hi Mohammad, what a surprise that you have already published Level 3 of sVB Kid Programmer. I am very happy about this special eBook for Turtle fans. Soon I will study your new Kindle edition and I am sure I will have a lot of fun.

Best regards...Gregor :)

**Comment by VBAndCs** [comment]
Your are welcome 🙂
I started the 4th book, and hope to finish it in 2 weeks. I will use the turtle with for loops in first chapter, then focus on arrays with for loops. I have an Idea to use arrays with the turtle, so it may play in the 2nd chapter too.
Thanks for helping evolve the sVB turtle. It is a powerful educational tool.
________________________________
